### ISSUE TITLE
Look into issues when copying old league into new ones. 

### ISSUE THREAD

**Issue by thomasbrewster** [cincyultimate/website#77]
Had issues creating Dayton Winter Indoor 2, when trying to copy it from the last one. I have two winter sessions so they may be part of the issue.

When I attempted the copy is said failed; don't remember exact error.

I worked around this by just creating a new fresh league and it's all setup now.

**Comment by kcin1018** [comment]
@thomasbrewster I tried to copy from all sorts of leagues and I am unable to reproduce this bug.

can you give me the info you entered when copying?

type
year
from league (exact name)

That way I can try to see what happened, thanks

**Comment by thomasbrewster** [comment]
Sure;

Well of course when I try to replicate the problem it works.
I've created the "2017 Winter Tuesday c78059f" league that can be deleted...

But I believe I might know what the problem was;
I think I was getting an "Error - invalid copy" - I note when I try to do a
copy and don't select a previous I get this screen

[image: Inline image 1]

I believe that is where I got stuck, I was on this screen and with the
"Copy from Previous" radio button selected in the above I didn't get the
option to select a previous session like I did when I did it correctly this
time.

Let me know if that doesn't make sense.

Regards,
Thomas
